modules/microshift-config-parameters-table.adoc

```diff
@@ -77,25 +77,46 @@ The following table explains {microshift-short} configuration YAML parameters an
 |`number`
 |By default, `etcd` uses as much memory as needed to handle the load on the system. However, in memory constrained systems, it might be preferred or necessary to limit the amount of memory `etcd` can to use at a given time.
 
+|`ingress.certificate.Secret`
+|`string`
+|A reference to a secret that contains the default certificate that is served by the ingress controller. When routes do not specify their own certificate, `certificateSecret` is used.
+
+The secret must contain the following keys and data:
+
+* `tls.crt`: certificate file contents
+* `tls.key`: key file contents
+
+If you do not set one of these values, a wildcard certificate is automatically generated and used. The certificate is valid for the ingress controller `domain` and `subdomains` fields, and the generated CA for the certificate is automatically integrated with the truststore for the cluster.
+
+Any certificate in use is automatically integrated in the {microshift-short} OAuth server.
+
+|`ingress.clientTLS`
+|`spec.clientTLS.clientCertificatePolicy`, `spec.clientTLS.ClientCA`, `AllowedSubjectPatterns`
+|`clientTLS` authenticates client access to the cluster and services; as a result, mutual TLS authentication is enabled. If you do not set a value, the client TLS is not enabled. The `clientTLS` parameter has the required subfields, `spec.clientTLS.clientCertificatePolicy` and `spec.clientTLS.ClientCA`.
```

Comment:
Why a comma separated list of the required fields here vs. the bullets used for others?
Not a big deal, the inconsistency just jumped out out me. Maybe it's because the text looks like a copy of the OpenShift docs.
https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/networking/networking-operators#nw-ingress-controller-configuration-parameters_configuring-ingress